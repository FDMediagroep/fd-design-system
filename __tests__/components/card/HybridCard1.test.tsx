import React from 'react';
import {
    HybridCard1,
    getCssClassNames,
} from '../../../src/components/card/HybridCard1';
import { render } from '@testing-library/react';

describe('HybridCard1', () => {
    test('should render correctly', () => {
        const { container, getByText } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                imageUrlS="https://images.example.com/imageS.png"
                imageUrlM="https://images.example.com/imageM.png"
                imageUrlL="https://images.example.com/imageL.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
            />
        );

        expect(getByText('This is the title')).toBeTruthy();
        expect(getByText('Ondernemen')).toBeTruthy();
        expect(getByText('This is the intro')).toBeTruthy();
        expect(getByText('This is the caption')).toBeTruthy();
        expect(container.querySelector('[href]')?.getAttribute('href')).toBe(
            'https://example.com/1/This is the title'
        );
    });

    test('should render is-read correctly', () => {
        const { container } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                isRead={true}
            />
        );
        expect(container).toMatchSnapshot();
    });

    test('should render variant-1 correctly', () => {
        const { container } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                variant="variant-1"
            />
        );
        expect(container).toMatchSnapshot();
    });

    test('should render themes correctly', () => {
        const { container: container1 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme1"
            />
        );
        const { container: container2 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme2"
            />
        );
        const { container: container3 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme3"
            />
        );
        const { container: container4 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme4"
            />
        );
        const { container: container5 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme5"
            />
        );
        const { container: container6 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme6"
            />
        );
        const { container: container7 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme7"
            />
        );
        const { container: container8 } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
                theme="theme8"
            />
        );
        expect(container1).toMatchSnapshot();
        expect(container2).toMatchSnapshot();
        expect(container3).toMatchSnapshot();
        expect(container4).toMatchSnapshot();
        expect(container5).toMatchSnapshot();
        expect(container6).toMatchSnapshot();
        expect(container7).toMatchSnapshot();
        expect(container8).toMatchSnapshot();
    });

    test('should have correct class name', () => {
        const { container } = render(
            <HybridCard1
                id="1"
                imageUrl="https://images.example.com/image.png"
                caption="This is the caption"
                intro="This is the intro"
                label="Ondernemen"
                title="This is the title"
                url="https://example.com/1/This is the title"
            />
        );

        expect(
            container.firstElementChild?.classList.contains(
                getCssClassNames()[0]
            )
        ).toBeTruthy();
    });
});
