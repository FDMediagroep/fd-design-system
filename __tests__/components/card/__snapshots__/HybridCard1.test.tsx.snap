// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HybridCard1 should render is-read correctly 1`] = `
<div>
  <article
    class="hybridCard1 isRead"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 1`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 2`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 3`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 4`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 5`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 6`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 7`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render themes correctly 8`] = `
<div>
  <article
    class="hybridCard1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;

exports[`HybridCard1 should render variant-1 correctly 1`] = `
<div>
  <article
    class="hybridCard1"
    data-variant="variant-1"
    id="1"
  >
    <a
      aria-label="This is the title"
      class="s__p-0 s__pt+4 s__pr+4 s__pl+4"
      href="https://example.com/1/This is the title"
    >
      <figure
        class="xs__m-0 s__mt-4 s__mr-4 s__mb+3 s__ml-4"
      >
        <picture>
          <img
            alt="This is the caption"
            src="https://images.example.com/image.png"
          />
        </picture>
        <figcaption>
          This is the caption
        </figcaption>
      </figure>
      <div
        class="teaserText xs__pt+3 xs__pr+2 xs__pb+2 xs__pl+3 s__p-0"
      >
        <div
          class="meta xs__mb+2"
        >
          <span
            class="prefix"
          >
            Ondernemen
          </span>
        </div>
        <div
          class="textContainer"
        >
          <h1
            class="heading serif xs xs__m-0"
          >
            This is the title
          </h1>
          <p
            class="intro body-text sans s xs__m-0 xs__mt+2"
          >
            This is the intro
          </p>
        </div>
      </div>
    </a>
    <footer
      class="footer footer xs__p-0 xs__pr+4 xs__pb+4 xs__pl+4"
    >
      <menu
        class="xs__m-0"
      >
        <a
          class="xs__mt+2"
        >
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              
   
              <path
                clip-rule="evenodd"
                d="M21 3.5V24.5L14 21L7 24.5V3.5H21ZM18.6667 5.83333H9.33333V20.7247L14 18.3913L18.6667 20.7247V5.83333Z"
                fill="#73655F"
                fill-rule="evenodd"
              />
              

            </svg>
            

          </span>
          <span
            data-bookmark="bookmark"
          >
            Bewaren
          </span>
        </a>
      </menu>
    </footer>
  </article>
</div>
`;
